website/views/static/bug.jade

```diff
@@ -0,0 +1,21 @@
+extends ./layout
+
+block vars
+  - var layoutEnv = env
+  - var menuItem = 'bug'
+
+block title
+  title Habitica &VerticalLine;&nbsp;
+    =env.t('communityBug')
+
+block content
+  .row
+    .col-md-12
+      .page-header
+        h1=env.t('communityBug')
+      p
+        =env.t('tryFirst')
+        button(onclick='localStorage.clear()') Clear local storage
```

Comment:
If it's going to be multiple things, it's probably best to extract this into an angular function.